src/stories/control/DarModeButton.tsx

```diff
@@ -0,0 +1,15 @@
+import { IconButton, IconButtonProps } from '@chakra-ui/react'
+import React from 'react'
+import { SunIcon, MoonIcon } from '@chakra-ui/icons'
+
+export interface DarkModeButtonProps extends IconButtonProps {
```

Comment:
Consider using the `useColorMode` hook instead to set/get colour mode properties:

https://chakra-ui.com/docs/styled-system/color-mode#usecolormode